x/slashing/types/expected_keepers.go

```diff
@@ -19,7 +19,6 @@ type AccountKeeper interface {
 type BankKeeper interface {
 	GetAllBalances(ctx sdk.Context, addr sdk.AccAddress) sdk.Coins
 	GetBalance(ctx sdk.Context, addr sdk.AccAddress, denom string) sdk.Coin
-	SetBalances(ctx sdk.Context, addr sdk.AccAddress, balances sdk.Coins) error
 	LockedCoins(ctx sdk.Context, addr sdk.AccAddress) sdk.Coins
 	SpendableCoins(ctx sdk.Context, addr sdk.AccAddress) sdk.Coins
 }
```

Comment:
strange, why do we copy the interface here?

Reply:
This has been a practice from long time with the keepers used in another modules.

Reply:
But here we can define the BankKeeper interface in the `bank` module and import it. I think it would be better. What do you think?

Reply:
it's how the SDK used to do things in the standard way. the scope of the PR was to get rid of usage of bank's unsafe API, so we didn't do anything else aside from that

Reply:
Yeah, I think this is out of the scope of this PR.